scripts/download.pl

```diff
@@ -70,27 +70,50 @@ ()
 	return undef;
 }
 
-sub download_cmd($) {
+sub download_cmd {
 	my $url = shift;
 	my $have_curl = 0;
-
-	if (open CURL, "curl --version 2>/dev/null |") {
+	my $have_aria2c = 0;
+	my $fn = shift;
```

Comment:
in theory we should be able to drop this and use directly $filename as it's global